core/object/method_bind.cpp

```diff
@@ -47,9 +47,11 @@ uint32_t MethodBind::get_hash() const {
 	}
 
 	hash = hash_murmur3_one_32(get_default_argument_count(), hash);
-	for (int i = 0; i < get_default_argument_count(); i++) {
-		Variant v = get_default_argument(i);
-		hash = hash_murmur3_one_32(v.hash(), hash);
+	for (int i = 0; i < get_argument_count(); i++) {
```

Comment:
There are two methods which have DEFVALs but no actual arguments. Those are `DirAccess::list_dir_begin` and `DisplayServer::is_touchscreen_available`.

If we want to avoid changing those hashes, I would suggest simply:
```cpp
for (const Variant &v : get_default_arguments()) {
	hash = hash_murmur3_one_32(v.hash(), hash);
}
```

Reply:
Hm. We probably shouldn't be including default arguments that aren't paired with actual arguments in the hash in the first place. And since we're already doing a mapping here, this actually seems like a good time to break those hashes. :-)

Locally, I tested removing the invalid default arguments, and the hashes (with the new calculation in this PR) remained stable, which I think is what we want. So, we can delete those `DEFVAL()`s at anytime, and if anyone adds any default arguments without arguments in the future, they won't be counted.